.gitignore

```diff
@@ -36,3 +36,9 @@ bower_components
 Thumbs.db
 
 .env
+
+# Bundled scripts
+website/assets/js/bundle.*.js
```

Comment:
this could be updated to just bundle.js, since chunkhash is no longer generated.